vertical-pod-autoscaler/vendor/k8s.io/metrics/pkg/client/external_metrics/fake/fake_client.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fake
```

Comment:
File modified in the `vendor` folder.
If we cannot amend the original module we should create something in our project.

Reply:
The file is not modified, it's just added to vendors because it's used now

Reply:
ok, got it